app/controllers/course_queues_controller.rb

```diff
@@ -27,6 +27,11 @@ def instructor_message
     @instructor_message = @course_queue.instructor_message
   end
 
+  # GET /course_queues/1/queues.json
```

Comment:
Thoughts on moving this to be an API on the course instead of on the queue?

Reply:
I've changed the endpoint to only return the other queues. It's fine to merge a queue with itself, but there isn't much of a use case. So, this removes it from the UI.

I think now that the endpoint is for other queues rather than all queues for the course, it makes sense to be under course_queues.